source/fundamentals/authentication.txt

```diff
@@ -0,0 +1,220 @@
+.. _csharp-authentication-mechanisms:
+
+=========================
+Authentication Mechanisms
+=========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to authenticate with MongoDB using the
+**authentication mechanisms** available in the {+mongo-community+}.
+Authentication mechanisms are processes by which the driver and server confirm
+the identity of a client to ensure security before connecting.
+
+You can use the following authentication mechanisms with the latest version
+of {+mongo-community+}:
+
+- :ref:`csharp-scram-sha-256`
+- :ref:`csharp-scram-sha-1`
+- :ref:`csharp-x509`
+
+To authenticate using ``GSSAPI/Kerberos`` or ``LDAP``, see the
+:ref:`csharp-enterprise-authentication-mechanisms` fundamentals page. For more 
+information on establishing a connection to your MongoDB cluster, see the
+:ref:`csharp-connect-to-mongodb`.
+
+Specify an Authentication Mechanism
+-----------------------------------
+
+You can specify your authentication mechanism and credentials when connecting to
+MongoDB using either of the following methods:
+
+- A **connection string**, also known as a **connection URI**, which is a string
+  that tells the driver how to connect to a MongoDB deployment and how to behave while
+  connected.
+
+- A factory method for the supported authentication mechanism, contained in the 
+  ``MongoCredential`` class.
+
+Mechanisms
+----------
+
+The following examples specify authentication mechanisms using the following
+placeholders:
+
+- ``<username>``: Your MongoDB username.
+- ``<password>``: Your MongoDB user's password.
+- ``<hostname>``: The network address of your MongoDB server, accessible by your client.
+- ``<port>``: The port number of your MongoDB server.
+- ``<authenticationDb>``: The MongoDB database that contains your user's authentication
+  data. If you omit this parameter, the driver uses the default value ``admin``.
+
+.. _csharp-authentication-default:
+
+Default
+~~~~~~~
+
+The default authentication mechanism setting uses one of the following
+authentication mechanisms, depending on which MongoDB versions your server supports:
+
+- ``SCRAM-SHA-256``
+- ``SCRAM-SHA-1``
+- ``MONGODB-CR``
+
+.. note::
+
+   MongoDB version 4.0 uses SCRAM as the default mechanism, and no longer
+   supports ``MONGODB-CR``. These versions automatically determine which
```

Comment:
nit: can the second part be formulated better, it sounds a bit not unrelated to the first?

Reply:
I agree - I just removed the second sentence, I don't think it was necessary